prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -52,10 +52,10 @@ import kotlin.system.exitProcess
 import kotlin.system.measureTimeMillis
 
 enum class TestStatus(val description: String) {
-    DRAFT("Experimental"), // Tests that are experimental
-    FAILS("Always fails"), // For tests that just always fail, and we haven't fixed the issue yet.
+    DRAFT("Experimental Test"), // Tests that are experimental
+    FAILS("(Always fails)"), // For tests that just always fail, and we haven't fixed the issue yet.
     LOAD("Load Test"),
-    SMOKE("Part of Smoke test"), // Only Smoke the Good Stuff.
+    SMOKE("Smoke Test"), // Only Smoke the Good Stuff.
 }
 
 class TestReportStream : CliktCommand(
```

Comment:
Add test with payloadName longer than database limit to ensure it doesn't break?
Add test with userName longer than database limit (63) to ensure it doesn't break? (unless that limit of 63 is enforced from Okta or something.